src/assets/layout/home.styl

```diff
@@ -1,3 +1,22 @@
 .home
   text-align center
-  
\ No newline at end of file
+  padding-top 50px
+
+.searcher
+  max-width 450px
```

Comment:
one of the best things to do is to order all the css properties alphabetically

Reply:
if you do, do it with all the css in this pr and after on the rest of the files of the project